editor/components/TableWrapper.tsx

```diff
@@ -119,9 +119,12 @@ export default function TableWrapper<T extends Record<string, unknown>>({
     hasAggregates: true,
   });
 
-  if (error) {
-    console.error(error);
-  }
+  // Log errors in an effect to avoid setState during render
+  useEffect(() => {
+    if (error) {
+      console.error(error);
+    }
+  }, [error]);
```

Comment:
This fixes React 18 strict mode violation where `console.error()` was called during render, triggering setState in the error overlay.